packages/plugin-errors/tests/index.test.ts

```diff
@@ -166,4 +168,13 @@ describe('errors plugin', () => {
       }
     `);
   });
+
+  it('supports generating custom names', () => {
+    const schema = createSchema(builderWithCustomErrorTypeNames)
+    expect(printSchema(schema)).toMatchSnapshot();
```

Comment:
thanks for updating test, I like the approach of wrapping the schema building logic in a function so you can build with different option!